### DISCUSSION TITLE
Customizing Node in the Network Component

### DISCUSSION THREAD

**Discussion by mahir-cognitiv** [plouc/nivo#1879] [Q&A] [answered]
I see there is a way to customize the Node in the network component, but there isnt any documentation or code examples to help me do so. Has anyone done this before?

**Comment by plouc** [comment, marked as answer]
Hi @mahir-cognitiv, there are examples in storybook:

- customizing nodes: https://nivo.rocks/storybook/?path=/story/network--custom-node
- customizing links: https://nivo.rocks/storybook/?path=/story/network--custom-link

Unfortunately, you cannot see the whole source code in the UI, but you can find it in the repo:

- nodes: https://github.com/plouc/nivo/blob/master/packages/network/stories/network.stories.tsx#L64
- links: https://github.com/plouc/nivo/blob/master/packages/network/stories/network.stories.tsx#L81

You might also want to have a look at the default components:

- [NetworkNode](https://github.com/plouc/nivo/blob/master/packages/network/src/NetworkNode.tsx)
- [NetworkLink](https://github.com/plouc/nivo/blob/master/packages/network/src/NetworkLink.tsx)

I hope this helps.

**Reply by mahir-cognitiv** [reply]
Thank you so much @plouc for your prompt response! This will really help :)
